### DISCUSSION TITLE
Make Taskbar right-click menu match the start-menu right-click menu. (Win 11 TaskBar)

### DISCUSSION THREAD

**Discussion by yodaluca23** [valinet/ExplorerPatcher#503] [Ideas]
Make TaskBar right-click menu match the start-menu right-click menu. The text in the taskbar right-click is smaller than the start right-click menu. please make it match up.
![image](https://user-images.githubusercontent.com/67206487/144616280-9f80bb07-e969-437a-9c16-569b2a6ac402.png)
![image](https://user-images.githubusercontent.com/67206487/144616290-15194b08-437f-47be-a0a4-f14318fa77cb.png)

**Comment by valinet** [comment]
Please do not open duplicates: https://github.com/valinet/ExplorerPatcher/discussions/501

I replied you there. As I said, you’re using the 11 taskbar, Microsoft does not offer an API to reproduce that menu style, as a matter of fact, they have one style there, one style for the taskbar icons and yet another style for the original right click menu of the taskbar (the one containing only “Taskbar settings”). Even they can’t get it straight, apparently. So yeah, without an API, there’s not much I can do. Even with one, it’s not something high on any priority list, that menu is mostly provided so that there is a consistent method to make it to EP “Properties”.

That being said, I will investigate at some point whether having the Start button open the legacy Win-X menu which looks like the taskbar right click menu is relatively easily doable, but I don’t have high hopes, all new UI development in Windows is mostly a fortress.

Thanks
